### PR TITLE
Dusk testing updates

### PR DESCRIPTION
**Pull request by alexstojda**
Duskt testing updates for acceptance tests. 

##### _Complete these items before requesting reviews_
- [x] Static Analysis Rules Validated
- [x] All commits follow naming conventions

### CODE REVIEW

**Comment by EvanDime** [comment]
I'll approve once i get an answer, but what is the difference between the dusk tag and using the id tag for identifying elements? I notice you use both. I have a PR that also adds more dusk but i'll update/fix things if i did something wrong.

**Comment by alexstojda** [comment]
> I'll approve once i get an answer, but what is the difference between the dusk tag and using the id tag for identifying elements? I notice you use both. I have a PR that also adds more dusk but i'll update/fix things if i did something wrong.

Both work, but the main reason I use a dusk tag is that the `id` must always be unique to the page, so I prefer to use dusk tags in some cases where I don't want to make sure the ID I choose isn't already in use on the page

**Review by EvanDime — DISMISSED** [review]

File: resources/js/Jetstream/DialogModal.vue [hunk 1]
```diff
@@ -1,48 +1,51 @@
 <template>
-    <modal :show="show" :max-width="maxWidth" :closeable="closeable" @close="close">
-        <div class="px-6 py-4">
-            <div class="text-lg">
-                <slot name="title">
-                </slot>
-            </div>
+  <modal :dusk="dusk" :show="show" :max-width="maxWidth" :closeable="closeable" @close="close">
```
Comment (a-drew) [inline resources/js/Jetstream/DialogModal.vue:2]: that one surprised me for a sec. was about tell you to put a better variable name till I realized this is a component

**Review by a-drew — APPROVED** [review]
tests looking good. bonus brownie points for converting the vue files to match our 2-tab coding standards

**Review by a-drew — APPROVED** [review]
Even more majestic than last time I approved 🎉

**Review by EvanDime — APPROVED** [review]

**Pull request merged**